platform/android/java/build.gradle

```diff
@@ -96,11 +96,11 @@ android {
 		]
 	}
 // No longer used, as it's not useful for build source template
-//	applicationVariants.all { variant ->
-//		variant.outputs.all { output ->
-//			output.outputFileName = "../../../../../../../bin/android_${variant.name}.apk"
-//		}
-//	}
+	applicationVariants.all { variant ->
```

Comment:
See the comment above :)

I don't think this change is related to joypad support on Android, so it should not be in this PR anyway.